app/react/Attachments/components/UploadSupportingFile.tsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/prop-types */
+
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
+import { Translate } from 'app/I18N';
+import { Icon } from 'UI';
+
+import { uploadAttachment } from '../actions/actions';
+import AttachmentsModal from './AttachmentsModal';
+
+interface UploadSupportingFileProps {
+  entityId: string;
+  entitySharedId: string;
+  storeKey: string;
+  progress?: any;
+}
+
+const UploadSupportingFile: React.FC<UploadSupportingFileProps> = props => {
+  const { entitySharedId, entityId, storeKey, progress } = props;
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const openModal = () => setModalOpen(true);
+  const closeModal = () => setModalOpen(false);
+
+  // Get percentage number
+  const getPercentage = progress.get(entitySharedId);
```

Comment:
Maybe is it better to change the name with the description of the comment?

Reply:
I just removed the comment